### PR TITLE
Turn off backlight when moving out

### LINKED ISSUE #151: Backlight should be turned off when moved out
To conserve the life of the backlight we should turn it off when it's out of the beam and on again when it's moved back in.

The PV to do this is BL03I-EA-BLIT-01:TOGGLE

### PR DESCRIPTION
**Pull request by noemifrisina**
Closes #151 

Link to Hyperion PR: [874](https://github.com/DiamondLightSource/hyperion/pull/874)

### CODE REVIEW

File: tests/devices/unit_tests/test_backlight.py [hunk 1]
```diff
@@ -14,3 +16,9 @@ def fake_backlight():
 def test_backlight_can_be_written_and_read_from(fake_backlight: Backlight):
     fake_backlight.pos.sim_put(fake_backlight.IN)
     assert fake_backlight.pos.get() == fake_backlight.IN
+
+
+def test_when_backlight_out_it_switches_off(fake_backlight: Backlight):
```
Comment (DominicOram) [inline tests/devices/unit_tests/test_backlight.py:21]: Nit: Could you add the opposite case too? Just so that we cover all paths

**Review by DominicOram — APPROVED** [review]
Looks good, thanks, minor nit pick but feel free to leave it

File: tests/devices/unit_tests/test_backlight.py (hunk 1 above)
Comment (noemifrisina) [reply tests/devices/unit_tests/test_backlight.py:21]: Done

Comment (noemifrisina) [reply tests/devices/unit_tests/test_backlight.py:21]: Done

**Review by DominicOram — APPROVED** [review]
New test looks good too, thanks.

**Pull request merged**